HACKING.md

```diff
@@ -46,10 +46,15 @@ the `snapd` project, please see [Contributing to snapd](./CONTRIBUTING.md).
 Build dependencies can automatically be resolved using `build-dep` on Ubuntu:
 
     cd ~/snapd
-    sudo apt-get build-dep .
+    sudo apt build-dep .
 
 Package build dependencies for other distributions can be found under the
-[./packaging/](./packaging/) directory.
+[./packaging/](./packaging/) directory. Eg. for Fedora use:
+
+    cd packaging/fedora
+    sudo dnf install -y rpmdevtools
+    sudo dnf install -y $(rpmspec -q --buildrequires snapd.spec)
+    sudo dnf install glibc-static.i686 glibc-devel.i686
```

Comment:
Are these required for unit tests to pass but not for building the snapd rpm? Else why not include in snapd.spec?

Reply:
That's the problem, I can't properly include them in the spec without breaking something. I've actually had a patch adding then to BuildRequires for x86_64 arch, but rpmbuild got unhappy and complained, even though dependencies were satisfied according to dnf, so I dropped the patch. Those packages are already included in the build images we use here and in Fedora infra.